docs/topics_5/rn-new-features.adoc

```diff
@@ -5,13 +5,60 @@
 
 This section describes the new features of the {ProductName} {DocInfoProductNumber}.
 
-Updated {WebName} interface::
+== Updated {WebName} interface
 The {WebName} has been redesigned with PatternFly 4. In addition to its general improved appearance, the {WebName} adds elements for project creation and management and updated transformation target icons.
 
-{ProductName} Operator for OpenShift Container Platform::
+== {ProductName} Operator for OpenShift Container Platform
 The current release includes the {ProductName} Community Operator, which installs the {WebName} on OpenShift Container Platform 4.5 and later.
 
-Greater support for migrating from Spring Boot to Quarkus::
-The current release provides greater support for migrating from Spring Boot to Quarkus with the addition of 22 rules.
-+
+== Greater support for migrating from Spring Boot to Quarkus
+The release contains numerous rules to support migrating from Spring Boot to Quarkus.
+
 The new rules focus on the Spring Boot APIs that are implemented in Quarkus Spring extensions, where Spring Boot dependencies can be replaced by equivalent Quarkus extensions. Spring artifacts that do not have Quarkus equivalents are also reported.
+
+Numerous other technologies, for which there are equivalent Quarkus extensions, are also identified by the new rules. These include:
+
+* Agroal
+* Amazon
+** Amazon DynamoDB
+** Amazon IAM
+** AWS Config
+** AWS Key Management Services (KMS)
+** AWS Lambda with HTTP APIs
+** AWS Lambda with AWS X-Ray
+** Amazon S3
+** Amazon Simple Email Service (SES)
+** Amazon Simple Notification Service (SNS)
+** Amazon Simple Queue Service (SQS)
+* Apache Avro
+* Apache Kafka
+* Artemis
+* Hibernate
+* Java Database Connectivity (JDBC)
+** Apache Derby (Java DB)
+** DB2
+** MariaDB
+** MySQL
+** PostgreSQL
+* JSONB
+* JSONP
+* Keycloak
+* Kubernetes
+* Narayana
+* OpenShift
+* RESTeasy
+* WildFly Elytron
+
+The new rules include rules for logging tools for which there are Quarkus extensions.
+
+To execute these rules, specify the target `quarkus`.
+
+=== Rules to support migrating to the most recent version of Quarkus
+
+This release contains rules to support migrating to the most recent release of Quarkus, as per the issues described in the link:https://github.com/quarkusio/quarkus/wiki/Migration-Guides[Quarkus Migration Guides].
+
+To execute these rules specify a target that reflects the Quarkus release version.
+
+For example, to migrate to latest release 1.11.x of Quarkus, specify a target of `quarkus1:11`.
```

Comment:
You can combine these sentences into a single sentence.

I'm not sure about "execute these rules". Are rules executed?